meteor-app/imports/ui/layouts/main/component.js

```diff
@@ -11,19 +11,22 @@ export default class MainLayout extends React.Component {
     const {
       store,
       body,
+      helpUrl,
     } = this.props;
+
     return (
       <div className="page layout-main">
         <div className="page__header">
-          <p>Some header</p>
```

Comment:
It's not unnecessary. It's used for taking up space for potential content so the header doesn't look absurdly narrow, especially on home page, which uses `main` layout.

See how home page looks like with this change:
![image](https://user-images.githubusercontent.com/1815174/28986689-a3cfc15e-792d-11e7-9fd6-37efc9dbe655.png)